src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+form {
+  margin-top: 20px;
+  box-sizing: border-box;
+}
+
+.input {
+  box-sizing: border-box;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+.inputBig {
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+  background-size: 19px;
+  background-position: 25px 24px;
+}
+
+.inputSmall {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 12px 14px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  font-weight: 900;
+}
+
+.inputBig:hover {
+  background-size: 19px;
+  background-position: 25px 24px;
+}
```

Comment:
lets unite such classes into one (`input`) and use it for big form

Reply:
Just remove .inputBig and transfer all it's properties to .input?